pkgs/development/tools/misc/loccount/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, lib, buildGoPackage, fetchgit }:
+buildGoPackage rec {
+  name = "loccount-${version}";
+  version = "20170213-${stdenv.lib.strings.substring 0 7 rev}";
+  rev = "7c45bc55394cd065f6a966110f5f8c3478998e95"; # 1.0 release
+
+  goPackagePath = "gitlab.com/esr/loccount";
+  excludedPackages = "tests";
+
+  src = fetchgit {
```

Comment:
so far only `fetchFromGitHub` is documented here: https://nixos.org/nixpkgs/manual/#idm140737317581056

You can use `nix-repl`:

```
nix-repl
Welcome to Nix version 1.11.7. Type :? for help.

nix-repl> :l <nixpkgs>
Added 7311 variables.

nix-repl> fetch<TAB>
fetchFromBitbucket  fetchFromSavannah   fetchTarball        fetchcvs            fetchgit            fetchgx             fetchpatch          fetchurl
fetchFromGitHub     fetchHex            fetchadc            fetchdarcs          fetchgitLocal       fetchhg             fetchsvn            fetchurlBoot
fetchFromGitLab     fetchMavenArtifact  fetchbower          fetchegg            fetchgitPrivate     fetchmail           fetchsvnrevision    fetchzip
fetchFromRepoOrCz   fetchNuGet          fetchbzr            fetchfossil         fetchgitrevision    fetchmtn            fetchsvnssh
```